roles/container-engine/containerd/defaults/main.yml

```diff
@@ -17,7 +17,7 @@ containerd_package: 'containerd.io'
 containerd_cfg_dir: /etc/containerd
 
 # Path to runc binary
-runc_binary: /usr/bin/runc
```

Comment:
why change ?

Reply:
This is not in the change set any more (use 'Files changed' tab).